### PR TITLE
BUG/ENH: Add support for numpy 1.9.x

### LINKED ISSUE #330: numpy FutureWarning raised in emperor.format
When running the jackknifed beta diversity command in the [QIIME 454 overview tutorial](http://qiime.org/tutorials/tutorial.html), the following warnings were raised within `emperor.format`:

```
$ jackknifed_beta_diversity.py -i otus/otu_table.biom -t otus/rep_set.tre -m Fasting_Map.txt -o jack -e 110
/Users/jairideout/.virtualenvs/qiime/lib/python2.7/site-packages/emperor/format.py:104: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  if coords_low != None and coords_high != None:
/Users/jairideout/.virtualenvs/qiime/lib/python2.7/site-packages/emperor/format.py:104: FutureWarning: comparison to `None` will result in an elementwise object comparison in the future.
  if coords_low != None and coords_high != None:
```

I have Emperor 0.9.5 and numpy 1.9.1 installed.

### PR DESCRIPTION
**Pull request by ElDeveloper**
.travis.yml has been updated to test for the latest available version of 
NumPy.

Fixes #330

### CODE REVIEW

**Comment by emperor-helper** [comment]
The test build for this pull request can be found here: http://emperor.colorado.edu/pull_331/make_emperor/

File: emperor/format.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     # write the values for all the ellipses
     js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
-    if coords_low != None and coords_high != None:
+    if is_empty(coords_low) and is_empty(coords_high):
```
Comment (Jorge-C) [inline emperor/format.py]: Is this equivalent? When `coords_low = None`, the previous code would result in a False but now it'd return True I think.

Comment (ElDeveloper) [inline emperor/format.py]: You are exactly right, I don't know what I was thinking about, thanks
for catching that!

File: emperor/format.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     # write the values for all the ellipses
     js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
-    if coords_low != None and coords_high != None:
+    if isinstance(coords_low, ndarray) and isinstance(coords_high, ndarray):
```
Comment (jairideout) [inline emperor/format.py]: Would it be simpler to test:

```python
if coords_low is not None and coords_high is not None:
```

?

Comment (Jorge-C) [inline emperor/format.py]: :+1:

Comment (ElDeveloper) [inline emperor/format.py]: Yeah, that does it, I'm having a slow day today ... as it turns out.

**Comment by Jorge-C** [comment]
Thanks @ElDeveloper!

**Comment by ElDeveloper** [comment]
Thanks!

**Pull request merged**